packages/react-components/react-avatar/src/components/AvatarGroup/AvatarGroup.tsx

```diff
@@ -5,9 +5,6 @@ import { useAvatarGroupStyles_unstable } from './useAvatarGroupStyles';
 import type { AvatarGroupProps } from './AvatarGroup.types';
 import type { ForwardRefComponent } from '@fluentui/react-utilities';
 
-/**
- * AvatarGroup component - TODO: add more docs
- */
```

Comment:
It'd probably be good to add a description here, but that can be done in a separate PR.